src/main/kotlin/cacophony/grammar/syntaxtree/Hierarchy.java

```diff
@@ -0,0 +1,25 @@
+package cacophony.grammar.syntaxtree;
+
+public class Hierarchy { //dont know how to implement it yet
```

Comment:
very cool you've done this work, I think all this will become clear after next classes on Tuesday